index.html

```diff
@@ -4214,24 +4542,26 @@ <h3>Parsing content</h3>
         must be <a id="#nfc-suspended">suspended</a>.
       </p>
     </section>
+
     <section> <h4>Permissions controls</h4>
       <p>
-        Making an <a>NFC tag</a> read-only must <a>obtain permission</a>, or
+        Making an <a>NFC tag</a> read-only MUST <a>obtain permission</a>, or
         otherwise fail.
       </p>
       <p>
-        Setting up listeners for reading <a>NFC content</a> should
+        Setting up listeners for reading <a>NFC content</a> SHOULD
         <a>obtain permission</a>.
       </p>
       <p>
-        The process of reading an <a>NDEF message</a> does not need to
+        Pushing <a>NFC content</a> to an <a>NFC peer</a> SHOULD
```

Comment:
Any reason why it is SHOULD and not MUST? If an NFC content can be pushed from peer to peer without user approval, this can be used for malware spread.

Reply:
Peer push requires user gesture of putting 2 devices in range, i.e. one or two persons is always involved, within an arms reach, though not necessarily knowing of each other (metro malware spread scenario). If the pushing user is malicious, will just okay the permission anyway. The other end should accept the push, though. If the page is malicious, obtaining permission before pushing is a MUST indeed.